example_secrets.tfvars

```diff
@@ -0,0 +1,9 @@
+# fake example secret data
```

Comment:
There is nothing in the instructions for configuring this file... we should probably also put in the instructions that if the user has already configured their workstation to use the AWS CLI/SDK, this file is unnecessary.